### DISCUSSION TITLE
Bug: Svelte syntax highlighting not working properly

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#13004] [Mobile] [answered]
I noticed that GitHub Linguist fully supports Svelte syntax highlighting, although in the (android) mobile app script and style is not syntax highlighted, see https://github.com/github/linguist/issues/5830#issuecomment-1072258062

![Screenshot_2022-03-18-11-05-28-151_com.github.android.jpg](https://user-images.githubusercontent.com/20150243/158983665-2bd607f2-d140-4480-af6f-4d990ea8a009.jpg)

![Screenshot_2022-03-18-11-05-38-330_com.github.android.jpg](https://user-images.githubusercontent.com/20150243/158983690-1dc8ceb2-87eb-4b57-9936-4fe1c63f70ce.jpg)

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @Zerotask! I can reproduce the bug, and I have created an internal issue to track it. I hope our engineers can work on a fix soon.

**Reply by camargo** [reply]
@aconsuegra Any update on this one? It looks like `.svelte` file highlighting is now broken in browsers as well (tested in latest Chrome, Firefox, and Safari). For example: https://github.com/sveltejs/kit/blob/master/sites/kit.svelte.dev/src/lib/search/SearchBox.svelte#L1

**Reply by danawoodman** [reply]
@aconsuegra from what I can tell, all `<script>` block contents are not highlighted at all, JS or TS:

TS:
![image](https://user-images.githubusercontent.com/157695/181083362-ba53911e-02c6-41c9-820b-97b0728249ad.png)

JS:
![image](https://user-images.githubusercontent.com/157695/181083424-02dc6752-d2e9-4871-b78a-f550b2b01863.png)

**Comment by camargo** [comment]
Any update on a fix for this?

**Reply by aconsuegra** [reply]
No, sadly the bug is still open. Sorry about the inconveniences.

**Reply by danawoodman** [reply]
Anything the community can do to help push this forward?

**Reply by aconsuegra** [reply]
You can follow up the current discussion in https://github.com/github/linguist/issues/5830. Svelte syntax highlighting was previously working in github.com but not on the Mobile apps, and it is now not working on any of the platforms.

**Comment by camargo** [comment]
This seems to be fixed in the browsers!
